api/serializers.py

```diff
@@ -1843,46 +1842,48 @@ def get_user_countries_regions(user):
 
     @staticmethod
     def get_local_unit_global_validators(user) -> List[int]:
-        data = list(
-            Permission.objects.filter(codename__startswith="local_unit_global_validator", group__user=user).values_list(
-                "codename", flat=True
-            )
-        )
-        global_validators = []
-        for code in data:
-            type_id = code.split("_")[-1]
-            global_validators.append((int(type_id)))
-        return global_validators
+        permission_codenames = Permission.objects.filter(
+            codename__startswith="local_unit_global_validator", group__user=user
+        ).values_list("codename", flat=True)
+        global_validators = {int(code.split("_")[-1]) for code in permission_codenames}
+        return list(global_validators)
 
     @staticmethod
-    def get_local_unit_region_validators(user) -> dict[str, list[int]]:
-        data = Permission.objects.filter(
+    def get_local_unit_region_validators(user) -> list[dict]:
+        permission_codenames = Permission.objects.filter(
             codename__startswith="local_unit_region_validator",
             group__user=user,
         ).values_list("codename", flat=True)
-
-        region_validators = defaultdict(set)
-
-        for code in data:
+        region_validators_map = {}
```

Comment:
No need to remove this.

```suggestion
        from collections import defaultdict
        region_validators_map = defaultdict(set)
```